locale/en/locale.cfg

```diff
@@ -63,38 +63,44 @@ map_info_sub_caption=    ..diggy diggy choo choo..
 map_info_text=The biters have catched the scent of fish in the cargo wagon.\nGuide the choo into the mountain and protect it as long as possible!\nThis however will not be an easy task,\nsince their strength and numbers increase over time.\n\nIn additon, the southern grounds collapse over time.\nStone bricks, concrete or other solid tiles, might improve the stability of the floor.\n\nDelve deep for greater treasures, but also face increased dangers.\nMining productivity research, will overhaul your mining equipment,\nreinforcing your pickaxe as well as increasing the size of your backpack.\n\nAs you dig, you will encounter impassable dark chasms or rivers.\nSome explosives may cause parts of the ceiling to crumble, filling the void, creating new ways.\nAll they need is a container and a well aimed shot.\n\nYou may find some supply goods, if you enter the wagon.\nGood luck on your journey!
 
 [chronosphere]
-map_info_main_caption=Chronosphere
+map_info_main_caption=C H R O N O S P H E R E
```

Comment:
```suggestion
map_info_main_caption=Chronosphere
```

Reply:
Not bothered either way, this was just in line with other Comfy scenarios